### PR TITLE
Added two new parameters ssl_cert and ssl_key

### PR DESCRIPTION
**Pull request by aleksvagachev**
##### SUMMARY
Added two new parameters ssl_cert and ssl_key to solve the issue: #424
Added an integration test to check the ssl connection

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Review by hunleyd — CHANGES_REQUESTED** [review]
Could you please add the fragment using docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment, so that our users will know about the fix

**Comment by aleksvagachev** [comment]
@hunleyd @Andersson007 Added changelog and integration test, do I need to add anything else?

**Review by hunleyd — APPROVED** [review]

**Comment by hunleyd** [comment]
@aleksvagachev lgtm .. @Andersson007 is on PTO this week, iirc but he'll look at it once he's back. i'd like another approval before i merge it (just to be safe)

**Comment by Andersson007** [comment]
@aleksvagachev if it's ready for review, could you please remove WIP from the title?

File: changelogs/fragments/1-postgres.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- postgresql - added new parameters ssl_cert and ssl_key for ssl connection (https://github.com/ansible-collections/community.postgresql/issues/424).
```
Comment (Andersson007) [inline changelogs/fragments/1-postgres.yml:2]:
```suggestion
- postgresql - added new parameters ``ssl_cert`` and ``ssl_key`` for ssl connection (https://github.com/ansible-collections/community.postgresql/issues/424).
```
to highlight them in rst a bit

File: plugins/doc_fragments/postgres.py [hunk 2]
```diff
@@ -54,6 +54,14 @@ class ModuleDocFragment(object):
       - If the file exists, the server's certificate will be verified to be signed by one of these authorities.
     type: str
     aliases: [ ssl_rootcert ]
+  ssl_cert:
+    description:
+      - Specifies the file name of the client SSL certificate.
+    type: path
```
Comment (Andersson007) [inline plugins/doc_fragments/postgres.py:60]:
```suggestion
    type: path
    version_added: '2.4.0'
```

File: plugins/doc_fragments/postgres.py [hunk 3]
```diff
@@ -54,6 +54,14 @@ class ModuleDocFragment(object):
...
+  ssl_key:
+    description:
+      - Specifies the location for the secret key used for the client certificate.
+    type: path
```
Comment (Andersson007) [inline plugins/doc_fragments/postgres.py:64]:
```suggestion
    type: path
    version_added: '2.4.0'
```

**Review by Andersson007 — COMMENTED** [review]
@aleksvagachev looks just great, thanks very much!

A couple of minor things from me

**Review by Andersson007 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**